### DISCUSSION TITLE
Severall questions before start

### DISCUSSION THREAD

**Discussion by OK1TK** [magicbug/Cloudlog#3021] [Q&A]
Hi guys, I use Logger32 for several years and now I found CloudLog. I was trying to get some answer for my questions but not 100% sure with some of them and the demo doesn't work now. At the beginning I would use the **Subscription** version and I would like to be sure that with this version I can:

- Does it work with clouLogOffilen in Windows as well ?
- Can I use CloudLog Aurora for automatted QSO upload from JTDX and is is possible to use it offline as well ?
- Can simply log QSO made as OK1TK/P, OK1TK/M and e.g. 9A/OK1TK/P ?
- Is it also possible to have our club station callsign with my account ?

Thank you very much for your help in advance.
Tomas

**Comment by magicbug** [comment]
Hi Tomas

> * Does it work with clouLogOffilen in Windows as well?

I'm not sure I don't use Cloudlogoffline

> * Can I use CloudLog Aurora for automatted QSO upload from JTDX and is is possible to use it offline as well ?

Cloudlog Aurora will send QSOs in real time to Cloudlog it doesn't however store them to upload once internet becomes available but its been asked for and on my dev plans at the moment you'd need to manually upload the ADIF.

> * Can simply log QSO made as OK1TK/P, OK1TK/M and e.g. 9A/OK1TK/P ?

You can create as many operating locations as you'd like these are grouped using Station Logbooks so that you can collate locations together for tracking awards progress.

> * Is it also possible to have our club station callsign with my account ?

Sure with hosted Cloudlog I don't limit the number of accounts just hope people keep it sensible so feel free to create.

**Comment by phl0** [comment]
> > * Does it work with clouLogOffilen in Windows as well?
> 
> I'm not sure I don't use Cloudlogoffline

Yes it does.

**Comment by OK1TK** [comment]
Thank you guys. I already set our club callsign and uploaded ADIF from Logger32. It is easy becasue there is only one location used. I need to figure out now how to upload my ADIF which contains more locations in one ADIF.

**Reply by magicbug** [reply]
Yes sadly at the moment we dont build the location profiles that being said if you have an ADIF with multiple locations and you email it to me I'll add it to my pile for working towards such a feature

**Reply by OK1TK** [reply]
Thank you, looks like I am somehow able to manage it in Logger32 with a liitle work. Then I can export different files.
